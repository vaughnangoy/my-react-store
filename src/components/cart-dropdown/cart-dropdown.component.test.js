import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';

import Header from './cart-dropdown.component';

describe('Cart Dropdown', () => {
    afterEach(cleanup);

    it('renders', () => {
        const { asFragment } = render(<Header />);
        expect(asFragment()).toMatchSnapshot();
    });
});
