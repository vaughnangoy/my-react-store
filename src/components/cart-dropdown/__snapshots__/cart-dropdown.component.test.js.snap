// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart Dropdown renders 1`] = `
<DocumentFragment>
  <div
    class="cart-dropdown"
  >
    <div
      class="cart-items"
    />
    <button
      class=" custom-button"
    >
      GO TO CHECKOUT
    </button>
  </div>
</DocumentFragment>
`;
